### DISCUSSION TITLE
Striplog sprint - `t21-hack-striplog`

### DISCUSSION THREAD

**Discussion by kwinkunks** [softwareunderground/transform-2021-hackathon#7] [Projects]
[`striplog`](https://github.com/agile-geoscience/striplog) needs some maintenance and there are lots of new features to add.

## Goals

- Open issues: there are plenty, including some new ones https://github.com/agile-geoscience/striplog/issues
- New functionality, e.g. see this thread: https://swung.slack.com/archives/C01FC2V9NVD/p1616330076013100
- `welly` enhancement to produce striplogs, e.g. add one or more boundary strength pickers to welly, resulting in striplogs

## People

- Brian Burnham
- Kieran Blacker
- Matt Hall

## Skills or interests

- Anyone is welcome to join! 
- It's a Python project, but we need documentation, tutorials, etc, etc. You don't have to be a programmer.
- We'd especially love to hear from people with use cases.
-

**Comment by 8556732** [comment]
Happy to be involved - do we perhaps want to rank the issues in order of priority?

**Reply by kwinkunks** [reply]
Yes, good idea... maybe we can use a Project or a Dicussion like this one in the `striplog` repo

**Comment by filippo82** [comment]
[T21 Hackathon Idea Mashup](https://softwareunderground.org/events/2021/4/7/t21-hackathon-idea-mashup) today (or tomorrow depending on your time zone) Wednesday April 14th at 16:00 UTC. Come to Slack to find the link for the meeting!
